.travis.yml

```diff
@@ -9,6 +9,8 @@ before_cache:
 
 before_install:
   - curl https://raw.githubusercontent.com/hmcts/reform-api-docs/master/bin/publish-swagger-docs.sh > publish-swagger-docs.sh
+  - sudo apt-get install jq
```

Comment:
does `sudo: required` not need to be declared?

Reply:
It seems not required as builds are passing and data are reported correctly.